server/graphql/v2/enum/PaymentMethodType.ts

```diff
@@ -0,0 +1,62 @@
+import { GraphQLEnumType } from 'graphql';
+
+import { PAYMENT_METHOD_SERVICE, PAYMENT_METHOD_TYPE } from '../../../constants/paymentMethods';
+import logger from '../../../lib/logger';
+import { PaymentMethod } from '../../../types/PaymentMethod';
+
+enum PaymentMethodTypeEnum {
+  CREDIT_CARD = 'CREDIT_CARD',
+  GIFT_CARD = 'GIFT_CARD',
+  PREPAID_BUDGET = 'PREPAID_BUDGET',
+  COLLECTIVE_BALANCE = 'COLLECTIVE_BALANCE',
+  PAYPAL = 'PAYPAL',
+  BANK_TRANSFER = 'BANK_TRANSFER',
+}
+
+export const PaymentMethodType = new GraphQLEnumType({
+  name: 'PaymentMethodType',
+  values: Object.keys(PaymentMethodTypeEnum).reduce((values, key) => {
+    return { ...values, [key]: {} };
+  }, {}),
+});
+
+export const getPaymentMethodType = ({ service, type }: PaymentMethod): PaymentMethodTypeEnum => {
+  if (service === PAYMENT_METHOD_SERVICE.STRIPE) {
+    if (type === PAYMENT_METHOD_TYPE.CREDITCARD) {
+      return PaymentMethodTypeEnum.CREDIT_CARD;
+    }
+  } else if (service === PAYMENT_METHOD_SERVICE.OPENCOLLECTIVE) {
+    if (type === PAYMENT_METHOD_TYPE.VIRTUALCARD) {
+      return PaymentMethodTypeEnum.GIFT_CARD;
+    } else if (type === PAYMENT_METHOD_TYPE.COLLECTIVE) {
+      return PaymentMethodTypeEnum.COLLECTIVE_BALANCE;
```

Comment:
`ACCOUNT_BALANCE` to match `PayoutMethod` and to hint that it's not only for Collectives anymore.